app/views/job_invocations/_tab_overview.html.erb

```diff
@@ -1,48 +1,22 @@
-<div id="status_chart">
-  <%= job_invocation_chart(job_invocation) %>
-</div>
-
-<div class="col-md-6 infoblock">
-  <h4><%= _('Target hosts') %></h4>
-  <% if job_invocation.bookmark.present? %>
-    <%= _('Bookmark') %> <%= job_invocation.bookmark.name %><br>
-  <% else %>
-    <%= _('Manual selection') %>
-  <% end %>
-  <%= 'using ' + _(Targeting::TYPES[job_invocation.targeting.targeting_type]) %>
-  <pre><%= job_invocation.targeting.search_query %></pre>
+<% template_invocations = job_invocation.pattern_template_invocations %>
+<div class='row'>
+  <div class="col-xs-6 col-sm-6 col-md-6">
+    <%= render :partial => 'card_results' %>
+  </div>
 
-  <%= _('Evaluated at:') %> <%= job_invocation.targeting.resolved_at %><br>
-  <% if job_invocation.pattern_template_invocations.size > 1 %>
-    <% job_invocation.pattern_template_invocations.each do |template_invocation| %>
-      <%= host_counter template_invocation.template.provider.humanized_name, ForemanTasks::Task::DynflowTask.for_action(Actions::RemoteExecution::RunHostJob).for_resource(template_invocation).uniq.size %>
+  <div class='col-xs-12 col-sm-6 col-md-6'>
+  <%= render :partial => 'card_target_hosts', :locals => { :job_invocation => job_invocation } %>
+  <% template_invocations.each do |template_invocation| %>
+    <% if template_invocation.effective_user %>
```

Comment:
It looks like the effective user is always displayed, however I can't show the 'total hosts' next to that. "Total hosts" displays the total number of hosts in the job invocation, but "effective user" is showed on a per-template invocation basis. I can put the "Total hosts" on top, and then the list of template invocation cards (effective user/user inputs). Usually there's only one.

Reply:
Are you sure? The template invocation IIRC is only one, since this is in fact pattern template invocation. I tried running on multiple hosts and I always get just one card, which is IMHO correct.

Reply:
I didn't know about `pattern template invocation`, I've just noticed the assignment on top, IMO it was confusing. That should only be one, unless ... you run a `JobInvocation` for 2 providers, in that case it's 2. We should really remove this asap.